h2o-py/h2o/model/model_base.py

```diff
@@ -1112,11 +1119,12 @@ def partial_plot(self, data, cols=None, destination_key=None, nbins=20, weight_c
             if num_2dpdp>0: # 2d pdp requested
                 axes3D = _get_mplot3d_pyplot("2D partial plots")
                 cm = _get_matplotlib_cm("2D partial plots")
-
+            figPlotted = 0  # indicated number of figures plotted
```

Comment:
this looks like a boolean

Reply:
Yeah, you are right.  Let me make it so.